features/04_aruba_api/filesystem/use_fixtures.feature

```diff
@@ -10,7 +10,7 @@ Feature: Use fixtures in your tests
     Given I use a fixture named "cli-app"
 
   Scenario: Use a fixture for your tests
-    Given a file named "spec/fixture_spec.rb" with:
+    Given a file named "spec/fixture_a_spec.rb" with:
```

Comment:
When you remove the changes to the timeouts (see my comment above) please keep the changes in this file.